lexicons/app/bsky/feed/post.json

```diff
@@ -38,6 +38,12 @@
             "type": "union",
             "refs": ["com.atproto.label.defs#selfLabels"]
           },
+          "tags": {
+            "type": "array",
+            "maxLength": 8,
+            "items": { "type": "string", "maxLength": 640, "maxGraphemes": 64 },
```

Comment:
maxLength v maxGraphemes is basically to allow for unicode stuff

i'm not sure we want to allow hashtags that are 64 emojis 😅

Which is to say: I think we'd be fine with a smaller max length here

Reply:
(altho tbh i'm not sure how much it really matters, what matters is that there is _some_ limit)

Reply:
I don't have an educated opinion on this, but keeping the max kinda roomy might be good for flexibility and creativity. Unlikely that 64 emoji hashtags will be used all that often.

Reply:
eh yeah seems fine 👍